setup.py

```diff
@@ -68,9 +79,7 @@ def run(self):
     author='HeikoHeiko',
     author_email='heiko@brainbot.com',
     url='https://github.com/raiden-network/raiden',
-    packages=find_packages(
-        exclude=["raiden.tests", "raiden.tests.*"]
-    ),
+    packages=find_packages(),
```

Comment:
what is the problem with including tests in a release? after all the tests are the guarantees about the correctness of the software we ship?